index.js

```diff
@@ -506,6 +513,23 @@ app.get('/Search',checkNotAuthenticated,(request,response) =>{
     })
   });
 
+  app.post('/Searchallrest',checkNotAuthenticated,(request,response) =>{
```

Comment:
what is this post request doing? We don't need this anymore as the two search pages are split up